python/pysmurf/client/tune/smurf_tune.py

```diff
@@ -3686,6 +3709,8 @@ def find_all_peak(self, freq, resp, subband=None, rolling_med=False,
             Whether to make a plot.
         save_plot : bool, optional, default True
             Whether to save the plot to self.plot_dir.
+        show_plot : bool, optional, defaullt False
```

Comment:
@eyyoung24 I think this doc string is using the wrong argument name. I think it should be `plot_phase` instead of `show_plot`?

Reply:
Added clarification on plot_phase.